packages/core/src/initialize.ts

```diff
@@ -12,17 +13,26 @@ export function initializeAgent(config: Config): Agent {
   const transports = initializeTransports(config);
   const api = initializeAPI(config, transports, metas);
 
+  const shutdown = () => {
+    shutdownInstrumentations(config);
+    transports.shutdown()
+    if (!agent.config.preventGlobalExposure) {
+      // @ts-ignore
```

Comment:
All `ts-ignore` directives should be accompanied by a message of why it is needed. E.g.: `// @ts-ignore: Bla bla bla bla`